### ISSUE TITLE
Figure out Organization vs SuperOrganization usage

### ISSUE THREAD

**Issue by nicholasserra** [SEED-platform/seed#752]
There seem to be two concepts of organization in the application. One is the traditional Organization model, which is in `seed.lib.superperms.orgs.models`. 

The second is from the package `django-organization` via `from organizations.models import Organization`. This package is included in INSTALLED_APPS, and is only used on the ImportRecord model as ImportRecord.organization. ImportRecord also has ImportRecord.super_organization.

My hunch is that ImportRecord.organization is unused and we can remove the entire package.

**Comment by nicholasserra** [comment]
This is turning out to be a blocker for 1.8, so going to take this on now.

**Comment by nicholasserra** [comment]
Any ideas from @nllong @mmclark @RDmitchell @StephenCzarnecki as to why django-organizations Organization model is wired to this ImportRecord model here?

https://github.com/SEED-platform/seed/blob/develop/seed/data_importer/models.py#L110

Almost finished removing it, but wondering what the deal was. Also would be interested to see if the db tables `data_importer_importrecord_organization` and `organizations_organization` are empty, as that's the for sure tell if it's unused.

**Comment by rebrownlbnl** [comment]
Yes, agree we don't want two different treatments of organizations.

I don't know the history about why a different package is used (or at
least mentioned) for file importing.

Consistency is good; as always, please try to improve the docs with
whatever you learn.

**Comment by nicholasserra** [comment]
So this will be a multi step migration patch. First I need to make a migration to remove the field in ImportRecord. Once that migration is deployed on all environments, I need to do a migration reset on the data_importer app to clean up history.

I'll need to figure out how these migrations will be deployed with @mmclark. The first migration needs to make it to production before the reset happens. So not sure if that requires it be in the feb release, that might be easiest.

**Comment by RDmitchell** [comment]
So this is all going to happen after the Feb release, in terms of getting these changes into develop and master, right? 

It seems like we will want to do a bunch of monkey (and unit?) testing on seeddev or seedtest (after the Feb release) when this change is in place.

**Comment by nicholasserra** [comment]
I need @mmclark's opinion on that. We can't reset migrations on the data_importer app until the new migration has been ran in all environments. He can probably do a hotfix to production.

**Comment by RDmitchell** [comment]
OK. We can definitely move this to a hotfix after the Feb release (which we might be able to release before the end of the month, depending on results of testing)

**Comment by StephenCzarnecki** [comment]
@nicholasserra 
No real insight on why here.  Looking at the history of that file that particular relationship has been there since the beginning.  As you have probably seen ImportRecord.organization isn't used anywhere, only ImportRecord.super_organization is used.  If I were to guess I'd say the many-to-many was for some intended functionality to let import records belong to multiple organizations.

ImportRecord.organization is probably not the only field like that.  When I was looking at the BuildingSnapshot model I found three other fields that do not seem to be used anywhere:  best_guess_canonical_building, best_guess_confidence and canonical_for_ds.  So I susped that, like these, ImportRecord.organization was either for some intended behavior that wasn't implemented or implemented behavior that was removed and these fields were left behind.

I can be more helpful regarding this question: "Also would be interested to see if the db tables data_importer_importrecord_organization and organizations_organization are empty on production"
using the production database dump from @mmclark on 2015-12-23 

I can tell you that organizations_organization is empty but orgs_organization is not.   There are three tables that start with organizations_ (organization, organizationowner, organizationuser).  All are empty.  There are also 3 fields that start with orgs_ (exportablefield, organization, organizationuser).  All are populated.

The data_importer_importrecord_organization table is also empty as are the following other data_importer_ tables:
- buildingimportrecord
- datacoercionmapping
- rangevalidationrule
- tablecolumnmapping
- validationoutlier
- validationrule

The following data_importer_ tables do have values
- importfile
- importrecord

**Comment by nicholasserra** [comment]
Awesome, thank you, those empty tables confirm my suspicions around this.

**Comment by mmclark** [comment]
The tables @nicholasserra mentioned above on production are indeed empty:

```
seed_prod=> \dt
seed_prod=> select count(*) from data_importer_importrecord_organization;
 count
-------
     0
(1 row)

seed_prod=> select count(*) from organizations_organization;
 count
-------
     0
(1 row)

```

**Comment by nicholasserra** [comment]
Assigning to @mmclark as he is the source of truth as to whether the necessary migrations have ran on production.

**Comment by RDmitchell** [comment]
@nicholasserra -- is there any manual testing I can do for this issue once it is merged into develop and on seedtest?

**Comment by nicholasserra** [comment]
No, I removed stuff that wasn't used anywhere, so nothing should be
different.
